backends/pkg/reconciler/eventmesh/handler_test.go

```diff
@@ -42,17 +41,17 @@ func TestBuildEventMesh(t *testing.T) {
 		{
 			name: "With 1 broker, 1 type, 1 trigger",
 			brokers: []*eventingv1.Broker{
-				testingv1.NewBroker("test-broker", "test-ns",
+				testingv1.NewBroker("test-broker", "default",
```

Comment:
All the namespace changes here can be reverted, right? As we don't hardcode the namespace as `default` anymore

Reply:
sure , i am on it .